core/new-gui/src/environments/environment.default.ts

```diff
@@ -31,7 +31,7 @@ export const defaultEnvironment = {
   /**
    * whether download execution result is supported
    */
-  downloadExecutionResultEnabled: false,
+  downloadExecutionResultEnabled: true,
```

Comment:
change it to default false, otherwise, it will affect developers who do not have such utility capability (i.e., no google auth config setup).